draft-ietf-tls-esni.md

```diff
@@ -495,7 +495,9 @@ are true:
 
 * Any referenced extension is missing in ClientHelloOuter.
 
-* "encrypted_client_hello" appears in OuterExtensions.
+* Any extension is referenced in OuterExtensions more than once.
+
+* "encrypted_client_hello" is referenced in OuterExtensions.
 
 * The extensions in ClientHelloOuter corresponding to those in OuterExtensions
   do not occur in the same order.
```

Comment:
Now that we have a section detailing this, maybe link to it in the paragraph below.